javascript/javascript1/week3/smart-ease.js

```diff
@@ -0,0 +1,149 @@
+const notes = [];
+function saveNote(content, id) {
+  notes.push({ content: content, id: id });
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+saveNote("Wash the dishes", 3);
+saveNote("Take a shower", 4);
+
+function getNote(id) {
+  for (let i = 0; i < notes.length; i++) {
+    if (!id || isNaN(id)) {
```

Comment:
Out of curiosity, why are you using `.isNaN`?

Reply:
In explanation of this exercise it says "If no id is specified or if the id is not a number, log out an error string."